doc/api/async_hooks.md

```diff
@@ -428,6 +438,13 @@ const server = net.createServer(function onConnection(conn) {
 });
 ```
 
+#### `async_hooks.registerTypeName(type)`
+
+* `type` {string} a new type of async resource
+* Returns {string} the type name to use
```

Comment:
Why not just return `undefined`? The way it is documented can be interpreted to imply some processing has been done on `type` before it is returned verbatim.